### ISSUE TITLE
camect-py incompatible with latest websockets

### ISSUE THREAD

**Issue by fracai** [camect/home-assistant-integration#16]
There's an [issue](https://github.com/camect/camect-py/issues/20) on the `camect-py` project where the `websockets` library changed the name of the `extra_headers` argument to `additional_headers`.
This breaks this integration for new installs that don't already have `websockets` installed.
There's a [pull request](https://github.com/camect/camect-py/pull/21/), but until that gets merged, I _think_ this could be resolved here by setting the websocket requirement to version 13.
```
  "requirements": [
    "camect-py==0.1.10"
    "websockets<=13"
  ],
```

Alternatively, edit `camect/__init__.py` to update the argument name until it's properly updated.

**Comment by chao-camect** [comment]
Yeah. We are holding PR 21 for now to avoid forcing users to upgrade to the latest version of websockets, which is still very new.
Feel free to create a PR to add the proposed change to require websockets<=13.
Thanks!

**Comment by sameer-git** [comment]
I was able to edit `camect/__init__.py` and fix up the argument as directed above and things are back to normal. FYI, I had to restart HA for the changes to take effect (reloading the integration did not work)

**Comment by epleypa** [comment]
Confirmed editing `camect/__init__.py` as directed worked for me as well

@chao-camect how long will camect/camect-py#21 be held?

**Comment by jaredkilgour** [comment]
> Confirmed editing `camect/__init__.py` as directed worked for me as well
> 
> @chao-camect how long will [camect/camect-py#21](https://github.com/camect/camect-py/pull/21) be held?

Hi all.  What is the edit to camect/__init__.py?   I'm experiencing this..  a net new install of the integration..  never had it working.  New to HA and no python experience.  Not clear to me what edit you two made.  Much appreciated.  Thanks!

**Comment by chao-camect** [comment]
I just merged the PR and uploaded a new version of the package, 0.2.1
